exampleplugins/5.2/SwagAttributeFilter/Subscriber/AttributeFilterSubscriber.php

```diff
@@ -36,11 +36,20 @@ public static function getSubscribedEvents()
         return [
             'Shopware_SearchBundleDBAL_Collect_Facet_Handlers' => 'registerFacetHandler',
```

Comment:
please use the service tags in the `services.xml` to register new facetHandler, conditionHandler and CriteriaRequestHandler. 
you should rewrite the whole plugin and the article

Reply:
Done